app/api/v1/endpoints/seasonal_forecast_pdf/schema.py

```diff
@@ -0,0 +1,9 @@
+from typing import Literal
+from pydantic import BaseModel
+
+country_name = Literal["zm", "mw"]
+
+
+class SeasonalForecastPDFParameters(BaseModel):
+    country: country_name = "mw"
+    district: str = "balaka"
```

Comment:
thanks for adding default options here, will make it easier for testing for sure